material-ui-chip-input/build.boot

```diff
@@ -61,6 +62,6 @@
      (minify :in  "cljsjs/material-ui-chip-input/development/material-ui-chip-input.inc.js"
              :out "cljsjs/material-ui-chip-input/production/material-ui-chip-input.min.inc.js")
 
-     (deps-cljs :name "cljsjs.material-ui-chip-input" :requires ["cljsjs.react" "cljsjs.material-ui"])
+     (deps-cljs :name "cljsjs.material-ui-chip-input" :requires ["react" "cljsjs.material-ui"])
```

Comment:
When using requires without `cljsjs.` prefix, the name should also skip the prefix.